packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CameraPlugin_Test.h

```diff
@@ -4,12 +4,12 @@
 
 // This header is available in the Test module. Import via "@import camera_avfoundation.Test;"
 
-#import "CameraPlugin.h"
+#import <camera_avfoundation/CameraPlugin.h>
 #import "FLTCam.h"
 #import "messages.g.h"
 
 /// APIs exposed for unit testing.
-@interface CameraPlugin ()
+@interface CameraPlugin () <FCPCameraApi>
```

Comment:
The plugin needs to implement the Pigeon API's `FCPCameraApi`. Previously the public header did this, but that meant the header file generated by Pigeon also needed to be public.

I switched this to follow `file_selector_ios`'s pattern.